rust-runtime/aws-smithy-runtime/src/client/orchestrator/http.rs

```diff
@@ -33,3 +37,25 @@ pub(crate) async fn read_body(response: &mut HttpResponse) -> Result<(), <SdkBod
 
     Ok(())
 }
+
+/// Marker value stored in the config bag to indicate that a response body should be redacted.
+#[derive(Debug)]
+pub struct SensitiveOutput;
```

Comment:
One last thing, I think this should be in aws-smithy-runtime-api alongside `LoadedRequestBody`.

Reply:
Thanks for suggesting a better place. Addressed in b4a68c4.